repos/fdbt-site/cypress_tests/cypress/e2e/myFares/myFares.cy.ts

```diff
@@ -28,6 +39,56 @@ describe('The my fares point to point products pages', () => {
         clickElementByText('Services');
         completeMyFaresPointToPointProductsPages();
     });
+    it('allows the user the edit point to point product passenger type', () => {
+        getHomePage();
+        clickElementById('account-link');
+        clickElementByText('Services');
+        editPassengerTypePointToPointPage();
+    });
+    it('allows the user the edit point to point product start date', () => {
+        getHomePage();
+        clickElementById('account-link');
+        clickElementByText('Services');
+        editStartDatePointToPointPage();
+    });
+    it('allows the user the edit point to point product end date', () => {
+        getHomePage();
+        clickElementById('account-link');
+        clickElementByText('Services');
+        editEndDatePointToPointPage();
+    });
+    it('allows the user the edit point to point product time restriction', () => {
+        getHomePage();
+        clickElementById('account-link');
+        clickElementByText('Services');
+        editTimeRestrictionPointToPointPage();
+    });
+    it('allows the user the edit point to point product fare triangle', () => {
+        getHomePage();
+        clickElementById('account-link');
+        clickElementByText('Services');
+        editFareTrianglePointToPointPage();
+        cy.get('[id=fare-triangle]')
+            .invoke('text')
+            .then((text) => {
+                const dateTextNew = text.split(' ')[1];
+                const [dayNew, monthNew, yearNew] = dateTextNew.split('/');
+                const newDate = new Date(+yearNew, monthNew - 1, +dayNew);
```

Comment:
monthNew is a string, can we do Number(monthNew) - 1 instead?

Reply:
and we may as well do the same for yearNew and dayNew also, instead of using +

Reply:
[08b984b](https://github.com/fares-data-build-tool/create-data/pull/641/commits/08b984b4576b8ad82aa6637843162d35f0c5c31d)